src/main/java/bio/terra/cli/command/resources/create/AiNotebook.java

```diff
@@ -177,7 +177,7 @@
     @CommandLine.Option(
         names = "--install-gpu-driver",
         description =
-            "If true, the end user authorizes Google Cloud to install a GPU driver on this instance")
+            "If true, the end user authorizes Google Cloud to install a GPU driver on this instance.")
```

Comment:
Maybe say that this is an extra charge, but then you have to indicate costs vs free everywhere. The phrase "authorizes Google Cloud to" is understood, since that happens with every option.

Reply:
I think this phrase is just roughly copied from the `gcloud notebooks instances create` command usage help:
```
       --install-gpu-driver
          Whether the end user authorizes Google Cloud to install a GPU driver
          on this instance. If this field is empty or set to false, the GPU
          driver won't be installed. Only applicable to instances with GPUs.
```